control-plane/rest/service/src/main.rs

```diff
@@ -205,11 +211,18 @@ async fn main() -> anyhow::Result<()> {
     utils::print_package_info!();
     let cli_args = CliArgs::args();
     println!("Using options: {:?}", &cli_args);
-    utils::tracing_telemetry::init_tracing(
-        "rest-server",
-        cli_args.tracing_tags.clone(),
-        cli_args.jaeger.clone(),
-    );
+
+    utils::tracing_telemetry::TracingTelemetry::builder()
+        .with_writer(FmtLayer::Stdout)
+        .with_style(FmtStyle::Pretty)
```

Comment:
Why do we have a separate `with_json` if we already have `.with_style`? Seems like Json is also a style.